docs/manual/developer_guide.adoc

```diff
@@ -1337,34 +1337,421 @@ Available low-level Jinja macros that can be used in Bash remediations:
+* Languages: Anaconda, Ansible, Bash, OVAL, Puppet
+
+sebool::
+* Checks values of SELinux booleans.
+* Parameters:
+** *seboolid* - name of SELinux boolean, eg. `cron_userdomain_transition`
+** *sebool_bool* - the value of the SELinux Boolean. Can be either `"true"` or `"false"`. If this parameter is not specified, the rule will use XCCDF Value `var_<seboolid>`. These XCCDF Values are usually defined in the same directory where the `rule.yml` that describes the rule is located. The *seboolid* will be replaced by a SELinux boolean, for example: `selinuxuser_execheap` and in the profile you can use `var_selinuxuser_execheap` to turn on or off the SELinux boolean.
+* Languages: Ansible, Bash, OVAL
+
+service_disabled::
+* Checks if a service is disabled. Uses either systemd or SysV init based on the product configuration in `product.yml`.
+* Parameters:
+** *servicename* - name of the service.
+** *packagename* - name of the package that provides this service. This argument is optional. If *packagename* is not specified it means the name of the package is the same as the name of service.
+** *daemonname* - name of the daemon. This argument is optional. If *daemonname* is not specified it means the name of the daemon is the same as the name of service.
+** *mask_service* - Check if the service should be masked to prevent it from running. This argument is optional. Implicit value is "true".
+* Languages: Ansible, Bash, OVAL, Puppet
+
+service_enabled::
+* Checks if a system service is enabled. Uses either systemd or SysV init based on the product configuration in `product.yml`.
+* Parameters:
+** *servicename* - name of the service.
+** *packagename* - name of the package that provides this service. This argument is optional. If *packagename* is not specified it means the name of the package is the same as the name of service.
+** *daemonname* - name of the daemon. This argument is optional. If *daemonname* is not specified it means the name of the daemon is the same as the name of service.
+* Languages: Ansible, Bash, OVAL, Puppet
+
+sshd_lineinfile::
+* Checks SSH server configuration items in `/etc/ssh/sshd_config`.
+* Parameters:
+** *parameter* - name of the SSH configuration option, eg. `KerberosAuthentication`
+** *value* - value of the SSH configuration option specified by *parameter*, eg. `"no"`.
+** *missing_parameter_pass* - If set to `"true"` the OVAL check will pass if the parameter is not present in `/etc/ssh/sshd_config`.
+* Languages: Ansible, Bash, OVAL
+
+sysctl::
+* Checks sysctl parameters. The OVAL definition checks both configuration and runtime settings and require both of them to be set to the desired value to return true.
+* Parameters:
+** *sysctlvar* - name of the sysctl value, eg. `net.ipv4.conf.all.secure_redirects`.
+** *datatype* - data type of the sysctl value, eg. `int`.
+** *sysctlval* - value of the sysctl value, eg. `'1'`. If this parameter is not specified, XCCDF Value is used instead.
```

Comment:
Do you mean xccdf variable?

Reply:
Yes, I mean a variable, but unfortunately, variables in XCCDF terminology are called XCCDF Values, and they're represented by `<xccdf:Value>` element :scream: